packages/daemon/src/app/OwnApp.js

```diff
@@ -126,6 +126,23 @@ export default class OwnApp extends AbstractApp {
 
   // Setters
 
+  set name(name: string) {
+    this._data.name = name
+  }
+
+  set contentsPath(path: string) {
+    this._data.contentsPath = path
+  }
+
+  editNextVersionNumber(version: string) {
+    const latestVersion = { ...this._versions[this._data.version] }
```

Comment:
Might be good to add 2 checks here:
- if the provided `version` is `this._data.version` this should be a no-op (simply return)
- if the provided `version` already exists is `this._versions`, it should error, as this might create some weird issues